AUTHORS.md

```diff
@@ -3,6 +3,7 @@
 - [Douglas Webb](https://github.com/bbew/), dmhwebb@gmail.com
 - [Florian Gössele](https://github.com/goessek/), goessek@posteo.de
 - [Kristijan Miklobusec](https://github.com/Sliverriver), kristijan80@web.de
+- [Lukas Gebhard](https://github.com/mr-kojo), freerunningapps@gmail.com
```

Comment:
Oh the authors file! I think you're one of the few people listed here that contributed code to karrot ;)

Reply:
In particular, @nicksellen is missing

Reply:
Yes, I wonder why we need this file at all. Git history is much more useful.